src/popup/router/components/AddressShortening.vue

```diff
@@ -58,22 +58,27 @@ export default {
 
 <style lang="scss" scoped>
 @use '../../../styles/variables';
+@use '../../../styles/typography';
```

Comment:
You need to make sure that in all places where it is used everything looks good.